frontend/src/modules/Shares/components/ShareBoxList.js

```diff
@@ -0,0 +1,489 @@
+    <>
+      <Helmet>
+        <title>Shares {tab} | data.all</title>
+      </Helmet>
+      <Box
+        sx={{
+          backgroundColor: 'background.default',
+          minHeight: '100%',
+          py: 1
+        }}
+      >
+        <Container maxWidth={settings.compact ? 'xl' : false}>
+          <Box
+            sx={{
+              mt: 2
+            }}
+          >
+            <Grid container spacing={2} xs={12}>
+              <Grid item md={3} xs={12}>
+                <Autocomplete
+                  id={'Status-' + tab}
+                  multiple
+                  fullWidth
+                  disableCloseOnSelect
+                  loading={loading}
+                  options={statusOptions}
+                  onChange={(event, value) =>
+                    handleFilterChange('Status', value)
+                  }
+                  renderOption={(props, option, { selected }) => (
+                    <li {...props}>
+                      <Checkbox
+                        icon={icon}
+                        checkedIcon={checkedIcon}
+                        style={{ marginRight: 8 }}
+                        checked={selected}
+                      />
+                      <ShareStatus status={option} />
+                    </li>
+                  )}
+                  renderInput={(params) => (
+                    <TextField
+                      {...params}
+                      label={'Status'}
+                      fullWidth
+                      variant="outlined"
+                    />
+                  )}
+                />
+              </Grid>
+              {!dataset && (
+                <Grid item md={3} xs={12}>
+                  <Autocomplete
+                    id={'Datasets-' + tab}
+                    multiple
+                    fullWidth
+                    disableCloseOnSelect
+                    loading={loading}
+                    options={datasets}
+                    getOptionLabel={(option) => option.label}
+                    onChange={(event, value) =>
+                      handleFilterChange('Datasets', value)
+                    }
+                    renderOption={(props, option, { selected }) => (
+                      <li {...props}>
+                        <Checkbox
+                          icon={icon}
+                          checkedIcon={checkedIcon}
+                          style={{ marginRight: 8 }}
+                          checked={selected}
+                        />
+                        {option.label}
+                      </li>
+                    )}
+                    renderInput={(params) => (
+                      <TextField
+                        {...params}
+                        label={'Datasets'}
+                        fullWidth
+                        variant="outlined"
+                      />
+                    )}
+                  />
+                </Grid>
+              )}
+              {!dataset && (
```

Comment:
That was my initial design, but for some reason it showed errors and was unable to render the page

Reply:
May work if add a empty parent elements `<> </>` wrapped around the 2 `<Grid>` components but not sure if it will change frontend UI formatting, either way it is very minor and still good as is